sandbox-templates/rehype-debug/src/index.js

```diff
@@ -0,0 +1,26 @@
+// ⚠️ Important! Please make sure the dependencies are up to date.
+// On code sandbox, you can refresh them in the Dependencies section (left-bottom)
+// On stackblitz, you can open the package.json file, update the versions,
+// then run npm install in the stackblitz terminal
+
+import { rehype } from "rehype";
```

Comment:
These files should be formatted. Example: https://github.com/remarkjs/.github/blob/1c6b1aa982f9de3742e09786ec0b6256c838b707/package.json#L27-L61

Reply:
updated